airflow/www_rbac/views.py

```diff
@@ -2067,7 +2067,7 @@ class ConnectionModelView(AirflowModelView):
                     'extra__google_cloud_platform__scope',
                     'extra__google_cloud_platform__num_retries',
                     'extra__grpc__auth_type',
-                    'extra__grpc__credentials_pem_file',
+                    'extra__grpc__credential_pem_file',
```

Comment:
Wow. Nice catch.

Reply:
@feluelle Reported it - I just fixed it :)